PixiEditor/ViewModels/SubViewModels/Main/FileViewModel.cs

```diff
@@ -195,23 +195,11 @@ private void Open(string path)
             }
             catch (CorruptedFileException ex)
             {
-                MessageBox.Show(ex.Message, "Failed to open file.", MessageBoxButton.OK, MessageBoxImage.Error);
+                NoticeDialog.Show(ex.Message, "Failed to open file.");
             }
             catch (OldFileFormatException)
             {
-                MessageBoxResult result = MessageBox.Show("This pixi file uses the old file format and is insecure.\nOnly continue if you trust the source of the file", "Old file format", MessageBoxButton.OKCancel);
-
-                if (result == MessageBoxResult.OK)
-                {
-                    try
-                    {
-                        OpenDocument(path, true);
-                    }
-                    catch (CorruptedFileException ex)
-                    {
-                        MessageBox.Show(ex.Message, "Failed to open file.", MessageBoxButton.OK, MessageBoxImage.Error);
-                    }
-                }
+                NoticeDialog.Show("This pixi file uses the old file format and can't be opened", "Old file format");
```

Comment:
Maybe "This .pixi file uses the old format, which is no longer supported and can't be opened." would be better